R/macro-package-checks.R

```diff
@@ -50,8 +50,8 @@ do_package_checks <- function(...,
       step_rcmdcheck(
         warnings_are_errors = !! enquo(warnings_are_errors),
         notes_are_errors = !! enquo(notes_are_errors),
-        args = !! enquo(args),
-        build_args = !! enquo(build_args),
+        args = args,
```

Comment:
Does it work if you leave this, and use `quo_is_null()` in `step_rcmdcheck()` ?

Reply:
Thanks, testing